other-docs/guides/launching-a-site-on-altis.md

```diff
@@ -0,0 +1,104 @@
+# Launching a Site on Altis
+
+The process for launching a site on Altis varies depending on the type of project. Nonetheless, there are common steps that need to be done no matter the context and this guide will walk you through what needs to be done in preparation for your site launch.
+
+## Planning
+
+Create a launch issue in the project repository. This issue should include:
+
+* Launch date
+* Feature freeze date
+* Code freeze date
+* Content freeze date (if migrating an existing site)
+* Team members needed for the deploy
+* Domains hosted on Altis
+* Domain(s) used for email
+* FROM address for each site in the network
```

Comment:
Are these not the same thing?

Reply:
Not necessarily. There are cases that I am currently aware of where there are multiple domains, but the FROM address for all of them is a single email address. For example if the domains were:

``` 
 - happyfunland.com
 - happyfunlandnews.com
 - happyfunlandgames.com
 - cruisehappyfun.com
 - etc
```

and they wanted the From email from all of those to be `noreply@happyfunland.com`.